lib/reverse_sentence.rb

```diff
@@ -1,6 +1,35 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: o(n^2)
```

Comment:
This is actually O(n), since you don't reverse the string n times, but only smaller words.  If you want to be technical it's O(n * m) where m is the length of a word, but since most words are pretty small  you can drop that term.